core-bundle/contao/dca/tl_content.php

```diff
@@ -612,14 +612,14 @@
 		'playerStart' => array
 		(
 			'inputType'               => 'text',
-			'eval'                    => array('rgxp'=>'natural', 'nospace'=>true, 'tl_class'=>'w25'),
-			'sql'                     => "int(10) unsigned NOT NULL default 0"
+			'eval'                    => array('maxlength'=>16, 'nospace'=>true, 'tl_class'=>'w25'),
```

Comment:
```suggestion
			'eval'                    => array('rgxp'=>'digit', 'maxlength'=>16, 'nospace'=>true, 'tl_class'=>'w25'),
```

Reply:
With this change you can still use start and stoptimes like:
`3`, `3.1` for your times.

Reply:
@zoglo What about `:` e.g. `hh:mm:ss` format?

Reply:
See https://github.com/contao/contao/pull/7465#issuecomment-2302998148